app/stores/DocumentsStore.ts

```diff
@@ -386,9 +386,32 @@ export default class DocumentsStore extends BaseStore<Document> {
       query,
     });
     invariant(res?.data, "Search response should be available");
+
     // add the documents and associated policies to the store
     res.data.forEach(this.add);
     this.addPolicies(res.policies);
+
+    // store a reference to the document model in the search cache instead
```

Comment:
I'm not super clear why this changed

Reply:
i'm storing reference to document model just like how `document_search` does and setting cache accordingly.  i'll remove extra props `ranking` `context` `id` which i'm returning. is this right?